admin/javascript/src/SilverStripeComponent.js

```diff
@@ -1,55 +0,0 @@
-/**
```

Comment:
This rename seems wrong. https://github.com/airbnb/javascript#naming--filename-matches-export